cabal.project

```diff
@@ -4,15 +4,19 @@ packages: language-plutus-core
           core-to-plc
           wallet-api
           plutus-th
+          plutus-use-cases
 optimization: 2
 constraints: language-plutus-core +development
            , wallet-api +development
            , core-to-plc +development
            , plutus-exe +development
            , plutus-core-interpreter +development
-tests: true
-benchmarks: true
-documentation: true
+-- tests: true
+-- benchmarks: true
+-- documentation: true
```

Comment:
Um, did you really mean to merge with these commented?

Reply:
Nope, that was a mistake.